src/pages/AddRecord/AddRecordInput.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react'
+import { INPUT_DETAILS } from '../../assets/constant/constant'
+import { CheckAllType, RecordType } from './AddRecord'
+
+type userProps = {
+  currentRecordType: RecordType
+  setCheckAllFilled: (value: any) => void
```

Comment:
```suggestion
  setCheckAllFilled: Dispatch<SetStateAction<CheckAllFilledType>>
```
이렇게 쓰려면 저 타입 임포트해야되긴 해여

Reply:
이부분 추가 설명이 필요합니다 ... 세인트!!

Reply:
이부분 추가 설명이 필요합니다 ... 세인트!!

> 모승!! 페이지에 기능이 정말 많은데 하나하나 다 만들어주시느라 수고하셨습니다!
> 
> 개인적인 생각입니다만 PR 단위가 너무 크다고 생각합니다! ㅠㅠ 그래서 코드 리뷰도 많아지고 기능도 많아서 조금 힘들었습니다. 다음에는 꼭 공용 컴포넌트로 뺄 수 있는 부분은 먼저 공용 컴포넌트 작업하고 다음 기능들을 구현하는 것이 어떨까 싶습니다!!
> 
> > input, textarea같은 기능별로 추상화했는데 추상화한다고 props로 전달되는 요소가 많음 즉 너무 과한 추상화였는지?
> 
> 3개정도는 props로 괜찮은 것 같습니다! checkAllFilled와 setCheckAllFilled를 props로 하는것이 좋을지는 잘 모르겠습니닷 세인트는 어떻게 생각하시나요? @Seongtaek-H
> 
> 추후 꼭 해야할 것 같은 부분
> 
> 1. AddCategory에서 `Chip` 공용 컴포넌트로 분리, AddRecordInput에서 `Input`공용 컴포넌트로 두개 컴포넌트 분리는 꼭 필요할 것 같습니다!
> 2. 세인트 말처럼 no-console eslint로 설정이 필요해보입니다!
> 
> -- **워터의 당부** 제가 코드가 깔끔한 것을 좋아하다보니 리뷰가 조금 디테일하게 들어갔을 것 같아요! 모승이 보고 너무 빡세거나 이정도는 넘어가도 될 것 같으면 언제든지 댓글 편하게 남겨주세요!

이게 왜 디테일한지 모르겠는 1인입니다 ... ㅋㅋㅋㅋㅋㅋ 더 해주셔도 상관없어요.
이건 제가 처음해서 어색했는데 다음에는 지라에 에픽을 하나파고 그 아래에 기능들을 최대한 쪼개서 나눠서 PR하는 방식으로 진행할게요 제 실수가 맞습니다.
공용 컴포넌트도 제가 훑어봤을떄는 재사용되지 않을거라 생각했는데 애매한거는 꼭 물어보고 컴포넌트화 하면 좋겠네요.
일단 이거는 머지되면 제가 다시 분리하는 작업을 진행할게요.
console은 제거를 할게요